pallets/communities/Cargo.toml

```diff
@@ -28,6 +30,9 @@ sp-core = { version = "21.0.0", git = "https://github.com/paritytech/substrate.g
 sp-io = { version = "23.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
 sp-runtime = { version = "24.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
 
+pallet-assets = { version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
```

Comment:
I think we can get rid of the version here (it applies for the rest of the packages)

Reply:
Gotcha